inspire/dojson/hep/fields/bd01x09x.py

```diff
@@ -95,15 +95,23 @@ def system_control_number2marc(self, key, value):
 
 
 @hep.over('report_number', '^037..')
-@utils.for_each_value
-@utils.filter_values
+#@utils.for_each_value
+#@utils.filter_values
 def report_number(self, key, value):
     """Source of Acquisition."""
-    return {
-        'primary': value.get('a', value.get('z')),
-        'arxiv_category': value.get('c'),
-        'source': value.get('9')
-    }
+    def get_value(value):
+        return {
+            'primary': bool(value.get('a')),
```

Comment:
Yeah, and can you `git grep` for `report_number.primary` lookups and change them to `report_number.value`. Same approach should in fact to applied to all keys that change their value or lookup key in this PR.